strawberry_django/optimizer.py

```diff
@@ -574,6 +574,15 @@ def _optimize_prefetch_queryset(
             else:
                 mark_optimized = False
 
+        if isinstance(field.type, type) and issubclass(field.type, Paginated):
+            pagination = field_kwargs.get("pagination")
+            qs = apply_window_pagination(
```

Comment:
is there a way to customise the queryset? should we document it?

Reply:
Hrm I haven't thought about that... good idea! Make it similar to the `Connection`